config/config.go

```diff
@@ -171,15 +171,36 @@ func newIcingaClient(c *SignaliloConfig, l logr.Logger) (icinga2.Client, error)
 		}
 	}
 
-	client, err := icinga2.New(icinga2.WebClient{
-		URL:               c.IcingaConfig.URL,
-		Username:          c.IcingaConfig.User,
-		Password:          c.IcingaConfig.Password,
-		Debug:             c.IcingaConfig.Debug,
-		DisableKeepAlives: c.IcingaConfig.DisableKeepAlives,
-		TLSConfig:         tlsConfig})
-	if err != nil {
-		return nil, err
+	var client *icinga2.WebClient
+
+	var tmp []string
+
+	for _, ur := range c.IcingaConfig.URL {
+		st := strings.Split(ur, ",")
+		for _, b := range st {
+			tmp = append(tmp, b)
+		}
+	}
+
+	c.IcingaConfig.URL = tmp
+
+	for _, url := range c.IcingaConfig.URL {
+		client, err = icinga2.New(icinga2.WebClient{
+			URL:               url,
+			Username:          c.IcingaConfig.User,
+			Password:          c.IcingaConfig.Password,
+			Debug:             c.IcingaConfig.Debug,
+			DisableKeepAlives: c.IcingaConfig.DisableKeepAlives,
+			TLSConfig:         tlsConfig})
```

Comment:
We don't really need to create a fresh client to test each URL. Calling `SetIcingaUrl()` for the first URL which passes `TestIcingaApi()` would suffice. 

Something like

```
client, err := icinga2.New(icinga2.WebClient{
	URL:               c.IcingaConfig.URL,
	Username:          c.IcingaConfig.User,
	Password:          c.IcingaConfig.Password,
	Debug:             c.IcingaConfig.Debug,
	DisableKeepAlives: c.IcingaConfig.DisableKeepAlives,
	TLSConfig:         tlsConfig})
if err != nil {
	// Failed to initialize the Icinga web client, return error
	return nil, err
}
for _, url := range c.IcingaConfig.URL {
	if err = client.TestIcingaApi(url); err == nil {
		client.SetIcingaUrl(url)
		break
	}
}
if err != nil {
	// We didn't find a valid API URL, return error
	return nil, fmt.Errorf("No valid Icinga API URL found")
}
return client, nil
```